docs/chart_best_practices/labels.md

```diff
@@ -0,0 +1,37 @@
+# Labels and Annotations
+
+This part of the Best Practices Guide discusses the best practices for using
+labels and annotations in your chart.
+
+## Is it a Label or an Annotation?
+
+An item of metadata should be a label under the following conditions:
+
+- It is used by Kubernetes to identify this resource
+- It is useful to expose to operators for the purpose of querying the system.
+
+For example, we suggest using `chart: NAME-VERSION` as a label so that operators
+can conveniently find all of the instances of a particular chart to use.
+
+If an item of metadata is not used for querying, it should be set as a label
```

Comment:
Right.  If its not used for querying, such as your example, should it not be an annotation instead?